<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equalizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="OpenFileButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
   <widget class="QSlider" name="ToneSlider">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>6</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ToneLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тон (БПФ)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FilePathEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>11</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ToneValueLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QToolButton" name="PlayBtn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/play.png</normaloff>:/buttons/play.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="SpeedValueLabel_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSlider" name="SpeedSlider_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>183</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Изменение скорости проигрывания &quot;стандартным&quot; способом, т.е. с искажениями тона</string>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QLabel" name="SpeedLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>183</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Скорость (Qt API)</string>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="StopBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/stop.png</normaloff>:/buttons/stop.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="PauseBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/pause.png</normaloff>:/buttons/pause.png</iconset>
    </property>
   </widget>
   <widget class="QSlider" name="trackSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>52</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelDuration">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>70</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-:--</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>136</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>100.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="SpeedLabel_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Громкость</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Настройка искажений</string>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>75</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Качество обработки</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ToneShifterQualityBox">
     <property name="geometry">
      <rect>
       <x>218</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>Качество влияет на скорость обработки. Чем выше качество - тем меньше скорость алгоритма</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="ResetBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сброс значений</string>
    </property>
   </widget>
   <widget class="QLabel" name="WaitCalcLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>73</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Идет обработка файла...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>OpenFileButton</zorder>
   <zorder>ToneSlider</zorder>
   <zorder>ToneLabel</zorder>
   <zorder>FilePathEdit</zorder>
   <zorder>ToneValueLabel</zorder>
   <zorder>PlayBtn</zorder>
   <zorder>SpeedValueLabel_2</zorder>
   <zorder>SpeedSlider_2</zorder>
   <zorder>SpeedLabel_2</zorder>
   <zorder>dial</zorder>
   <zorder>StopBtn</zorder>
   <zorder>PauseBtn</zorder>
   <zorder>trackSlider</zorder>
   <zorder>labelDuration</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>SpeedLabel_3</zorder>
   <zorder>ResetBtn</zorder>
   <zorder>WaitCalcLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="buttons.qrc"/>
 </resources>
 <connections/>
</ui>
